README.md

```diff
@@ -1 +1,94 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top">SET UP LINTERS AND CREATE PULL REQUEST</a>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="https://zbahati.github.io/Hello-Microverse/">visit website</a>
+
+**[Hello Microverse]**
+is a web application that help us to set up the linters for the project that we will use to test the project.
+
+#### this project have  one section that  is h1 with the message Hello Microverse.
+
+<!-- Features -->
+
+### Key Features <a name="https://zbahati.github.io/Hello-Microverse/">header-section features</a>
+
+
+- **[Test Linters of HTML and CSS ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+> Add a link to your deployed project.
+
```

Comment:
- [ ] As requested by the previous reviewer, please remove the default comment like 

>  Add a link to your deployed project

>  Describe how a new developer could make use of your project. (line 28)

from your README. They were used in the template to explain to the user the purpose of each section. You can go ahead and remove it from your README